sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/CreateTables.java

```diff
@@ -123,6 +123,8 @@ private void possibleCreateTable(
         // every thread from attempting a create and overwhelming our BigQuery quota.
         DatasetService datasetService = bqServices.getDatasetService(options);
         if (!createdTables.contains(tableSpec)) {
+          tableReference.setTableId(
```

Comment:
<!--thread_id:cc_139875266_t; commit:101ff5301f10803cae2c4c22f786f0b3c6d0a7c9; resolved:1-->
<!--section:context-quote-->
> **jkff** wrote:
> It is odd that we do this only in this branch. Can we do it at line 117? Or perhaps even better, move line 117 and 118 to before line 126, since that's where they are used?

<!--section:body-->
Done.